src/content/docs/setup/configuration/content-delivery-network.mdx

```diff
@@ -679,6 +679,39 @@ Ensure that the base URL change is propagated to Catalog Service. You can do thi
 
 ![Commerce base URL](@images/implementation/commerce-base-url.png)
 
-## Debugging
+## Troubleshooting
+
+<Aside type="note" title="It's confusing - we get it">
+Fastly VCL and CDN configuration can be a bit overwhelming at first. Here's a collection of troubleshooting steps that might help you if you have problems.
+</Aside>
+
+### Fastly API Quick Reference
+
+With your Fastly API Token you can make requests against the Fastly API for information or data that may be otherwise unaccessible. You can find your API Token in your Commerce Admin view under **Stores** > **Settings** > **Configuration** > **Advanced** > **System** > **Full Page Cache** > **Fastly Configuration**.
```

Comment:
```suggestion
With your Fastly API Token you can make requests against the Fastly API for information or data that may be otherwise inaccessible. You can find your API Token in your Commerce Admin view under **Stores** > **Settings** > **Configuration** > **Advanced** > **System** > **Full Page Cache** > **Fastly Configuration**.
```